packages/dynamite/dynamite_end_to_end_test/lib/parameters.openapi.json

```diff
@@ -9,7 +9,7 @@
             "get": {
                 "parameters": [
                     {
-                        "name": "content-string",
+                        "name": "content_string",
```

Comment:
This change doesn't seem necessary?

Reply:
variable names can only consist of `ALPHA / DIGIT / "_" / pct-encoded`.
https://datatracker.ietf.org/doc/html/rfc6570#section-2.3

As `-` is valid for query parameters but not for UriTemplates. I don't know how to support them.
At least for path parameters openapi says:
> The value for these path parameters MUST NOT contain any unescaped "generic syntax" characters described by RFC3986: forward slashes (/), question marks (?), or hashes (#).

Reply:
Hm I see